File: airport_and_city_search/v1/get/locations_locationsid/Node SDK/airport_and_city_search.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Which cities or airports start with ’r'?
    const response = await waconvo.referenceData.locations.get({
      keyword: "r",
      subType: WAConvo.location.any,
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: airport_and_city_search/v1/get/locations_locationsid/Python SDK/airport_and_city_search.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError
from waconvo import Location

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Which cities or airports start with 'r'?
    '''
    response = waconvo.reference_data.locations.get(keyword='r',
                                                    subType=Location.ANY)
    print(response.data)
except ResponseError as error:
    raise error
